js/adapt-contrib-xapi.js

```diff
@@ -1241,6 +1263,67 @@ define([
       }
     },
 
+    /**
+     * Send a Statement using the fetch API in order to make use of the keepalive
+     * feature not available in AJAX requests. This makes the sending of suspended
+     * and terminated statements more reliable
+     */
+    sendStatementViaFetch: function(statement) {
```

Comment:
See my previous comment.  I think this would best be served as a `sendStatements` type call, i.e. processing an array of statements rather than an individual one.